docs/githubconnector/demoscenario.md

```diff
@@ -0,0 +1,96 @@
+# Usage of GitHub Connector <!-- omit in toc -->
```

Comment:
This file should be moved, it does apply to the overall scenario (and not only Github Connector) - best place for this doc would be one directory above: `docs/`.

Reply:
The technical side of this doc looks fine, please ask TWs for a language review.